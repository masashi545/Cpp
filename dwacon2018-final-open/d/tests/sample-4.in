10
983102606006243867
653718290103598600
364611268624595444
114746989192634390
81304291426411017
931878752092058491
395809284336497545
633900034071891379
895817108011279740
92661392530626177
