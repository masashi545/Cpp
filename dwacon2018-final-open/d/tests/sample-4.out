893653300
150104699
232570112
922156483
361136690
103094234
245249617
912578727
399641917
820143308
